src/components/Filters/components/ConnectedFilterControl/ConnectedFilterControl.tsx

```diff
@@ -35,9 +35,7 @@ interface State {
   proxyButtonsWidth: ComputedProperty;
 }
 
-export const FILTER_FIELD_MIN_WIDTH = 150;
-
-export const FILTER_FIELD_CUSTOM_PROPERTY = '--textfield-min-width';
```

Comment:
This was never used anywhere, but because it was exported TS didn't warn us of that.